### DISCUSSION TITLE
Record audio using microphone in wavesurfer.js then create regions or split that recording into regions

### DISCUSSION THREAD

**Discussion by shahbazg4045** [katspaugh/wavesurfer.js#3172] [General]
Hi, 
I have create a html table with two columns (words, recordings). Now when I record the voice using API it will create a blob file then split that blob file into regions mean detect silence part and separate into regions. 

But I want to show recording in-front of each words that I record using microphone on setInterval say 2 second. 

When 2 second gone its will split that region and display in html table.

**Comment by katspaugh** [comment]
Sounds like a cool project. Thanks for sharing. Feel free to reach out for help if you have any concrete questions.
